src/Network/Transport/TCP.hs

```diff
@@ -24,8 +24,6 @@ module Network.Transport.TCP
   , createTransportExposeInternals
   , TransportInternals(..)
   , EndPointId
-  , encodeEndPointAddress
-  , decodeEndPointAddress
```

Comment:
It's not necessary. Encoding/decoding an address just doesn't strike me as something that a typical user of network-transport-tcp will need to do, so I export them only from `Internal`.